Src/Tracer.Logger.EurekaLog.pas

```diff
@@ -44,6 +44,17 @@ procedure TEurekaLogRefCountTracerLog.DoLogStackTrace;
 
 procedure TEurekaLogRefCountTracerLog.LogLine(
   const CallStack: TEurekaBaseStackList; const Index: Integer);
+
+  function GetAddressOf(var AVariable): string;
+  begin
+    Result := IntToHex( Integer( Pointer( @AVariable ) ), 8 );
+  end;
+
+  function PointerToHex(var AVariable): string;
```

Comment:
I'm not sure why you inserted this. As I don't have Eurekalog installed I cannot test. But there is the function PointerToHex in Tracer.Logger.Tools.pas (which is 64bit aware). Your new function doesn't really make sense to me. Isn't Info.Location.Address not already a pointer?